plugins/grid-bookmark/src/index.ts

```diff
@@ -70,10 +70,6 @@ export default class extends Plugin {
                     const regions = bookmarkWidget.bookmarkedRegions
                     if (regions.length !== 0) {
                       self.navTo(regions.at(-1))
-                      session.notify(
```

Comment:
seemed to perhaps be unnecessary to give the success notify, open to opinion though